bindings/java/src/org/sleuthkit/datamodel/SleuthkitJNI.java

```diff
@@ -395,7 +396,29 @@ public synchronized String currentDirectory() {
 			 */
 			@Deprecated
 			public void run(String[] imageFilePaths) throws TskCoreException, TskDataException {
-				run(null, imageFilePaths);
+				run(null, imageFilePaths, 0);
+			}
+			
+			/**
+			 * Starts the process of adding an image to the case database.
+			 * Either AddImageProcess.commit or AddImageProcess.revert MUST be
+			 * called after calling AddImageProcess.run.
+			 *
+			 * @param deviceId       An ASCII-printable identifier for the
+			 *                       device associated with the image that
+			 *                       should be unique across multiple cases
+			 *                       (e.g., a UUID).
+			 * @param imageFilePaths Full path(s) to the image file(s).
+			 *
+			 * @throws TskCoreException if a critical error occurs within the
+			 *                          SleuthKit.
+			 * @throws TskDataException if a non-critical error occurs within
+			 *                          the SleuthKit (should be OK to continue
+			 *                          the process)
+			 */
+			@Deprecated
```

Comment:
Why deprecate this method? I don't think we should deprecate this method, it seems helpful to treat auto-detect as the default vs. requiring a user to enter the somewhat artificial sector size of zero argument. Do you have a counterargument?